couch/CHANGELOG.md

```diff
@@ -1,16 +1,12 @@
 # CHANGELOG - couch
 
-2.2.0 / Unreleased
+2.1.0 / Unreleased
 ==================
-### Changes
-
-* [UPDATE] Update auto_conf template to support agent 6 and 5.20+. See [#860][]
-
-2.1.0 / 2017-10-5
-=================
 
```

Comment:
Note: I changed this because https://github.com/DataDog/integrations-core/pull/793 actually updated the changelog incorrectly - the date should have been `Unreleased`